scripts/kulture.js

```diff
@@ -0,0 +1,127 @@
+
+      const finalPosition = adressHeight ?  (titleHeight + (adressHeight / 2)) : ((titleHeight / 2) - expectedElem.outerHeight() / 2);
+      expectedElem.css({top: finalPosition + 'px'});
+    });
+  }
+
+  function tranformLibraryToAccordion(pageElemRef) {
+    var libraryAccordion = pageElemRef.find('.layout-wrapper .primary-content .pane-weekly-opening-hours');
+
+    libraryAccordion.find('h2.pane-title').replaceWith(function() {
+      return $("<a/>", {html: $(this).html()});
+    });
+
+    libraryAccordion.find('.panel-pane-inner').attr("class", "accordion-wrapper");
+
+    var link = libraryAccordion.find('.accordion-wrapper > a');
+    link.attr("class", "accordion-control collapsed");
+    link.attr("data-toggle", "collapse");
+    link.attr("role", "button");
+    link.attr("data-target", "#library-details-accordion");
+    link.attr("aria-controls", "#library-details-accordion");
+
+    var accordionBody = libraryAccordion.find('.accordion-wrapper > .pane-content');
+    accordionBody.attr("id", "library-details-accordion");
+    accordionBody.attr("class", "collapse");
+  }
+
+  function addCollapseToList(listContainerRef) {
+    listContainerRef.find('.panel-pane-inner').addClass("accordion-wrapper");
+
+    listContainerRef.find('.panel-pane-inner > h2.pane-title').replaceWith(function() {
+      return $("<a/>", {html: $(this).html()});
+    });
+
+    var link = listContainerRef.find('.panel-pane-inner > a');
+    link.attr("class", "accordion-control collapsed");
+    link.attr("data-toggle", "collapse");
+    link.attr("role", "button");
+    link.attr("data-target", "#category-filter-accordion");
+    link.attr("aria-controls", "#category-filter-accordion");
+
+    link.append('<span class="arrow"></span>');
+
+    var accordionBody = listContainerRef.find('.accordion-wrapper > .pane-content');
+    accordionBody.attr("id", "category-filter-accordion");
+    accordionBody.attr("class", "accordion-filter-body collapse");
+  }
+
+  function addCollapseToNewsFilter(listContainerRef) {
+    console.log(listContainerRef);
+    listContainerRef.find('.select-list').addClass("accordion-wrapper");
+
+    listContainerRef.find('.select-list h2.list-title').replaceWith(function() {
+      return $("<a/>", {html: $(this).html()});
+    });
+
+    var link = listContainerRef.find('.select-list > a');
+    link.attr("class", "sub-menu-title accordion-control collapsed");
+    link.attr("data-toggle", "collapse");
+    link.attr("role", "button");
+    link.attr("data-target", "#news-library-filter-accordion");
+    link.attr("aria-controls", "#news-library-filter-accordion");
+
+    link.append('<span class="arrow"></span>');
+
+    var accordionBody = listContainerRef.find('.accordion-wrapper > .list-items');
+    accordionBody.attr("id", "news-library-filter-accordion");
+    accordionBody.attr("class", "list-items accordion-filter-body collapse");
+  }
+
+
+  $(document).ready(function () {
```

Comment:
Here can be used drupal behaviours: https://www.drupal.org/node/756722#behaviors